.changeset/soft-stingrays-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/auth': patch
```

Comment:
This isn't a patch; it changes database structure for anyone using `auth` with a field path other than `password` for the secret field.

We can decide to ship this as a breaking change, or we can fix the bug in the other direction (by making `getPasswordResetSchema.ts` generate the field path dynamically based on the `secretField` config when the `tokenType` is `passwordReset`)